### DISCUSSION TITLE
Search users by description

### DISCUSSION THREAD

**Discussion by iHazzu** [tweepy/tweepy#1785] [Questions] [answered]
Hello, I'm trying to build a query to search users through the `search_users` function. But this `q` parameter of the function seems to only support searching by username, since when I try to search for another field, I always get an empty list as response.
```python
import tweepy
import keys

auth = tweepy.OAuthHandler(keys.CONSUMER_KEY, keys.CONSUMER_SECRET)
auth.set_access_token(keys.ACCESS_TOKEN, keys.ACCESS_TOKEN_SECRET)

api = tweepy.API(auth)

users = api.search_users(q="description=vampire", count=10)
print(users)

# []
```
[This site](https://tweeplesearch.com/) has a Twitter account search tool by bio. Would this be possible through the API provided by Twitter?

**Comment by Harmon758** [comment, marked as answer]
[GET users/search](https://developer.twitter.com/en/docs/twitter-api/v1/accounts-and-users/follow-search-get-users/api-reference/get-users-search), the endpoint that `API.search_users` uses:
> Provides a simple, relevance-based search interface to public user accounts on Twitter.

I believe this includes results based on descriptions. You'll have to filter them out yourself, as there are no other Twitter API endpoints to search for users.
